test/eventing.bash

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+function checkout_knative_eventing {
+  checkout_repo 'knative.dev/eventing' \
+    "${KNATIVE_EVENTING_REPO}" \
+    "${KNATIVE_EVENTING_VERSION}" \
+    "${KNATIVE_EVENTING_BRANCH}"
+}
+
+function checkout_knative_eventing_operator {
+  checkout_repo 'knative.dev/eventing-operator' \
+    "${KNATIVE_EVENTING_OPERATOR_REPO}" \
+    "${KNATIVE_EVENTING_OPERATOR_VERSION}" \
+    "${KNATIVE_EVENTING_OPERATOR_BRANCH}"
+}
+
+function run_knative_eventing_tests {
+  (
+  local exitstatus=0
+  logger.info 'Running eventing tests'
+
+  checkout_knative_eventing
+
+  go_test_e2e -timeout=90m -parallel=1 ./test/e2e \
```

Comment:
I'm curious if `-parallel=1` is necessary here. Does anyone have experience with higher parallelization? I'm thinking that by adding these two test suites the overall CI time will be quite long. So better try to reduce it where we can.

Reply:
we have `12` in the `release-next` branch:

https://github.com/openshift/knative-eventing/blob/master/openshift/e2e-tests-openshift.sh#L147

(like upstream)

Reply:
Yup. I put it ad it was in 0.13 branch. For next release the test section is quite different, not only the parallel have been changed.